scripts/eqtl_hail_batch/calculate_ld.py

```diff
@@ -0,0 +1,64 @@
+"""Read in correlation results and claculate ld values"""
+
+import hail as hl
+import click
+import pandas as pd
+
+
+@click.command()
+@click.option('--input-path', help='A path prefix of where to input files are located')
+def query(input_path):
+    """calculate LD values"""
+
+    hl.init(default_reference='GRCh38')
+
+    tmp_dir = input_path.replace(input_path.split('/')[2], input_path.split('/')[2] + '-tmp')
+    mt_path = f'{tmp_dir}/genotype_table.mt/'
+    mt = hl.read_matrix_table(mt_path)
+    mt = mt.annotate_rows(
+            global_bp=hl.locus(
+                mt.locus.contig, mt.locus.position
+            ).global_position(),
+        )
+    # if running on entire genome, concatenate all significant
+    # correlation matrices together, then read in as one df
+    # example: https://stackoverflow.com/questions/20906474/import-multiple-csv-files-into-pandas-and-concatenate-into-one-dataframe
+    significant_snps_path = f'{input_path}correlation_results.csv'
+    significant_snps_df = pd.read_csv(
+        significant_snps_path, sep=' ', skipinitialspace=True
+    )
+    t = hl.Table.from_pandas(significant_snps_df)
+    # only keep rows whose FDR is < 0.05
+    # t = t.filter(t.fdr < 0.05)
+    print(f'Printing table: {t.show()}')
+    t = t.key_by('global_bp')
+    # filter mt to positions which are in significant_snps table
+    mt = mt.filter_rows(hl.is_defined(t[mt.global_bp]))
+    # add row index to be able to remap
+    mt = mt.add_row_index()
+    # turn matrix into table and save, in order to reference row idx
+    mt_path = f'{input_path}significant_snps.mt'
+    mt.write(mt_path)
+    # perform ld calculation
+    ld = hl.ld_matrix(mt.GT.n_alt_alleles(), mt.locus, radius=2e6)
+    # only calculate the upper triangle 
+    ld = ld.sparsify_triangle()
+    table = ld.entries()
+    # filter out entries with an LD score less than 0.2
```

Comment:
I'm happy to remove it, only previously the majority of rows were taken up by very low LD values. in GTEx, the minimum they use is 0.1, but in a data meeting I think we settled on 0.2. Very happy to change this though and not use any filter, as, like you said, we're only dealing with significant SNPs now. Regardless, if we set some minimum value, I agree that we should make the clear in the browser. 
Just a side note, the `calculate_ld.py` script will need further refining as I test it once all the significant SNPs come in, so this can be a conversation we have with the Daniels, or I can test and see how many results I actually get and how large the output matrix is. @daniaki or @danielreti , do you have any thoughts?

Reply:
Testing without the filter first sounds like a good idea, then if we need to slim the results down we can add a filter of 0.1 like GTEx.

Reply:
Sounds great, I'll test that one first :)